metadata-ingestion/src/datahub/ingestion/transformer/dataset_transformer.py

```diff
@@ -39,6 +46,55 @@ class DatasetTagsTransformer(DatasetTransformer, metaclass=ABCMeta):
     def aspect_name(self) -> str:
         return "globalTags"
 
+    @staticmethod
+    def merge_with_server_global_tags(
```

Comment:
As per our style guide, these should be standalone methods instead of staticmethods

maybe we should move them to a transformer helpers file?